QuizApp/App/Screens/Quiz/QuizListViewModel.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 final class QuizListViewModel {
 
-    var categories: [CategoryViewModel] = [.sport, .movies, .music, .geography]
+    var categories: [Category] = [.all, .sport, .movies, .music, .geography]
```

Comment:
You now have two models in the UI layer for the same thing - Category and CategoryViewModel. The latter is enough